CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 2.2.2 (Under active development)
 
+### App Center Analytics
+
+#### WPF/WinForms
+
+* **[Fix]** Fixed wrong naming system product name and system manufacturer.
```

Comment:
Please be more explicit here. The fix is to avoid sending some default placeholders, not the "wrong" values.

@guperrot @achocron any suggestions?

Reply:
Fix reporting the correct device models?